app/main.py

```diff
@@ -2,0 +1,50 @@
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name: str = name
+        self.skills: list = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["JavaScript", "HTML", "CSS"]
+
+    def create_awesome_web_page(self) -> str:
+        print(f"{self.name} is creating a webpage...")
+        return "<h1>Hello world</h1>"
+
+
+class BackendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["Python", "SQL", "Django"]
```

Comment:
Instead of directly assigning a new list to `self.skills`, use `self.skills.extend(['Python', 'SQL', 'Django'])` or `self.skills += ['Python', 'SQL', 'Django']` to add these skills to the inherited list. This follows the checklist guideline to use `extend()` or `+=` for adding multiple values to a list .